src/nanoemoji/paint.py

```diff
@@ -115,17 +118,25 @@ def breadth_first(self) -> Generator[PaintTraverseContext, None, None]:
     def children(self) -> Iterable["Paint"]:
         return ()
 
+    def gettransform(self) -> Optional[Affine2D]:
```

Comment:
why did you type the return value as `Optional[Affine2D]` instead of just Affine2D?